GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/VFSAppDelegate.m

```diff
@@ -1,30 +1,41 @@
 #import "VFSAboutWindowController.h"
 #import "VFSAppDelegate.h"
+#import "VFSMessageListener.h"
+#import "VFSNotificationDisplay.h"
+#import "VFSForGitNotification.h"
 #import "VFSProductInfoFetcher.h"
 #import "VFSStatusBarItem.h"
-#import "VFSUserNotification.h"
+#import "VFSNotificationDisplay.h"
 
 @interface VFSAppDelegate ()
 
 @property (weak) IBOutlet NSWindow *Window;
 @property (strong) VFSStatusBarItem *StatusDisplay;
+@property (strong) VFSMessageListener *messageListener;
+
+- (void)displayNotification:(NSDictionary *_Nonnull)messageInfo;
 
 @end
 
 @implementation VFSAppDelegate
 
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification
 {
-    // TODO : Create a message listener object and
-    // start listening for incoming notification
-    // requests.
+    self.messageListener = [[VFSMessageListener alloc]
+        initWithSocket:NSTemporaryDirectory()
+        callback:^(NSDictionary *messageInfo)
+        {
+            [self displayNotification:messageInfo];
+        }];
+    
+    [self.messageListener startListening];
     
     VFSProductInfoFetcher *productInfoFetcher =
```

Comment:
nit: what's going on with the whitespace here? Can we try getting Xcode to make the whitespace conform to the usual standard?